### PR TITLE
Escape sequence detection improvements and enabling CI

### PR DESCRIPTION
**Pull request by BanzaiMan**

### CODE REVIEW

File: spec/log/deansi.coffee [hunk 1]
```diff
@@ -136,7 +136,7 @@ describe 'deansi', ->
       '''
 
       lines = progress 5, (ix, count, curr, total) ->
-        end = if count == 100 then ", done.\e[K\n" else "   \e[K\r"
+        end = if count == 100 then ", done.[K\n" else "   [K\r"
```
Comment (BanzaiMan) [inline spec/log/deansi.coffee]: There are invisible ESC (`\033`) embedded here in front of `[`.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+language: node_js
+
+before_install: ruby -S gem install bundler
```
Comment (joshk) [inline .travis.yml]: isn't bundler already installed?

File: spec/log/deansi.coffee [hunk 3]
```diff
@@ -147,7 +147,7 @@ describe 'deansi', ->
         end = if count == 100 then ", done.\n" else "   \r"
         [ix + 12, "Resolving deltas: #{count}% (#{curr}/#{total})#{end}"]
 
-      lines = [[0, "Cloning into 'jsdom'...\n"], [1, "remote: Counting objects: 13358, done.\e[K\n"]].concat(lines)
+      lines = [[0, "Cloning into 'jsdom'...\n"], [1, "remote: Counting objects: 13358, done.[K\n"]].concat(lines)
```
Comment (BanzaiMan) [inline spec/log/deansi.coffee]: Invisible ESC (`\033`) exists here also.

File: .travis.yml (hunk 2 above)
Comment (BanzaiMan) [inline .travis.yml]: It wasn't clear to me. :-) I can remove it, of course.

**Pull request merged**